dora-boots/bytecode_dump.dora

```diff
@@ -86,14 +86,14 @@ class BytecodeDumper(let bc: BytecodeFunction): BytecodeVisitor {
         println(" ${r1}, ${gid.value}");
     }
 
-    fun emitFctVoid(name: String, fid: FctId, r1: BytecodeRegister, cnt: Int) {
+    fun emitFctVoid(name: String, fid: FctId, cnt: Int) {
```

Comment:
I think it's time to also get rid of the `count` argument, not in this PR though.